### PR TITLE
Adding Lithuanian language "lt" support

### PR DESCRIPTION
**Pull request by Drasius2**
Adding Lithuanian language "lt" support

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,32 +133,33 @@ Supported languages
...
+| Korean               | `ko`     |
+| Lithuanian           | `lt`     |
+| Norwegian            | `no`     |
+| Polish               | `pl`     |
+| Portuguese           | `pt`     |
+| Russian              | `ru`     |
+| Spanish              | `es`     |
+| Swedish              | `sv`     |
+| Turkish              | `tr`     |
+| Ukrainian            | `uk`     |
```
Comment (EvanHahn) [inline README.md]: Any reason why this table changed?

**Comment by EvanHahn** [comment]
You can ignore @houndci-bot's comments.

Other than my comment about the readme, this looks great!

File: README.md (hunk 1 above)
Comment (Drasius2) [inline README.md]: Damn... There is aditional space at the end. I was added a short version of "lt" language - "shortLt". And the table was to short. But later I decided to remove it (the "shortLt") and forgot to shrink the table.

**Comment by EvanHahn** [comment]
_Ačiū!_ This is now part of `humanize-duration@3.6.0`.

**Pull request merged**